Cue/app/tabs/library/CardListViewRow.js

```diff
@@ -53,39 +92,92 @@ export default class CardListViewRow extends React.Component {
       flag = <Image style={styles.flagIcon} source={CueIcons.indicatorFlag} />
     }
 
-    let row = (
-      <View style={styles.container}>
+    let deleteContainer
+    let handleContainer
+    if (this.props.editing) {
+      let deleteIcon = (
+        <View style={styles.deleteContainer}>
+          <View>
```

Comment:
Umm you know, sometimes you just want an extra `View`, you know?

(It was probably there for layout, but the layout works fine without it, so...)